linux_sql/scripts/psql_docker.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+#Start Docker if not done so already
+if systemctl status docker || systemctl start docker
+then
+  echo "Docker is Active"
+  echo $1
+
+fi
+
+#Create Container Clause
+if [ $1 = "create" ]
```

Comment:
please review shell style.

you need capture and assign argument to variable name at the beginning of the script. this is not readable

e.g.
username=$1
password=$2

and use $username instead of $1

Reply:
fix this issue for all arguments

Reply:
changes were fixed